content/en/cypherpunk/timothy-may/the-crypto-anarchist-manifesto.md

```diff
@@ -0,0 +1,20 @@
+
+La tecnologia per questa rivoluzione - che sarà sicuramente una rivoluzione sociale ed economica - esiste in teoria da un decennio. I metodi si basano sulla crittografia a chiave pubblica, sui sistemi di prova interattivi a conoscenza zero e su vari protocolli software per l'interazione, l'autenticazione e la verifica. Finora l'attenzione si è concentrata sulle conferenze accademiche in Europa e negli Stati Uniti, conferenze monitorate attentamente dalla National Security Agency. Ma solo di recente le reti informatiche e i personal computer hanno raggiunto una velocità tale da rendere le idee effettivamente realizzabili. E i prossimi dieci anni porteranno un'ulteriore velocità sufficiente a rendere le idee economicamente fattibili e sostanzialmente inarrestabili. Reti ad alta velocità, ISDN, scatole a prova di manomissione, smart card, satelliti, trasmettitori in banda Ku, personal computer multi-MIPS e chip di crittografia attualmente in fase di sviluppo, saranno alcune delle tecnologie abilitanti.
+
+Lo Stato cercherà ovviamente di rallentare o fermare la diffusione di questa tecnologia, adducendo preoccupazioni per la sicurezza nazionale, l'uso della tecnologia da parte di spacciatori ed evasori fiscali e timori di disintegrazione della società. Molte di queste preoccupazioni saranno valide; la cripto-anarchia permetterà di commerciare liberamente segreti nazionali e materiali illeciti e rubati. Un mercato computerizzato anonimo renderà possibile persino i ripugnanti mercati degli omicidi e delle estorsioni. Diversi elementi criminali e stranieri saranno utenti attivi di CryptoNet. Ma questo non fermerà la diffusione della cripto-anarchia.
+
+Così come la tecnologia della stampa ha alterato e ridotto il potere delle corporazioni medievali e la struttura del potere sociale, allo stesso modo i metodi crittografici altereranno fondamentalmente la natura delle corporazioni e dell'interferenza dei governi nelle transazioni economiche. In combinazione con i mercati emergenti dell'informazione, la cripto-anarchia creerà un mercato liquido per tutto il materiale che può essere trasformato in parole e immagini. E proprio come un'invenzione apparentemente minore come il filo spinato ha reso possibile la recinzione di vasti ranch e fattorie, modificando per sempre i concetti di terra e di diritti di proprietà nel West di frontiera, così anche la scoperta apparentemente minore di un'arcana branca della matematica diventerà il tagliafili che smantella il filo spinato intorno alla proprietà intellettuale.
```

Comment:
Consiglio sostituzione di `apparentemente minore` in `[…], apparentemente di scarsa rilevanza, […]`